tests/misc/test_extra_group_pkgs_can_be_installed.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import pytest
+
+from lib.commands import SSHCommandFailed
+from packaging import version
+from subprocess import run, CalledProcessError
+
+def install_failed(pkg_name, h):
+    """Check if a package can be installed or not.
+
+    Returns True if the package cannot be installed, False otherwise.
+    It saves the yum state before trying to install a package and restore
+    it after.
+    """
+    failed = False
+    h.yum_save_state()
+
+    try:
+        h.yum_install([pkg_name])
+    except SSHCommandFailed:
+        failed = True
+
+    h.yum_restore_saved_state()
+    return failed
+
+def test_extra_group_packages_can_be_installed(host):
+    v = version.parse(host.inventory['PRODUCT_VERSION'])
```

Comment:
I don't think it's necessary to parse the version into a version object, and it's bit risky to then rely on its conversion back to string to generate the URL.

You can probably just keep the original string all along.

Reply:
But the URL is not the same for 8.2 and 8.3.

Reply:
you can ge the version from `host.xcp_version` though.
So either read the string from the inventory and don't parse it. Or use the one stored in the host object and stringify it.
I would simply read it and not use `version`.